### PR TITLE
Add TomTom MyDrive Connect Latest

### PR DESCRIPTION
**Pull request by fturcheti**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **updating a cask**:

- [ ] `sha256` changed but `version` stayed the same ([what is this?][version-checksum]).  
      I’m providing public confirmation below.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-drivers/pulls
[closed issues]: https://github.com/caskroom/homebrew-drivers/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/tomtom-mydrive-connect.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  sha256 :no_check
+
+  url 'https://cdn.sa.services.tomtom.com/static/sa/Mac/MyDriveConnect.dmg'
+  name 'TomTom MyDrive Connect'
+  homepage 'https://www.tomtom.com/mydrive-connect/'
+
+  pkg 'MyDriveConnect.mpkg'
+
+  uninstall pkgutil:    [
+                          'com.tomtom.TomTomMyDriveConnectHelper',
```
Comment (commitay) [inline Casks/tomtom-mydrive-connect.rb]: @fturcheti This doesn't seem to be a `pkg` that is installed?

Comment (fturcheti) [inline Casks/tomtom-mydrive-connect.rb]: This is the first time I'm creating a cask... so I don't know if I fully understand your question. Are you asking about `com.tomtom.TomTomMyDriveConnectHelper`? Should it be removed from the `pkgutil` directive?

Comment (commitay) [inline Casks/tomtom-mydrive-connect.rb]: >Should it be removed from the `pkgutil` directive?

Possibly. Where did you get `com.tomtom.TomTomMyDriveConnectHelper` from?

Comment (fturcheti) [inline Casks/tomtom-mydrive-connect.rb]: It's a helper app – `TomTomMyDriveConnectHelper.app` – that lives inside `TomTom MyDrive Connect.app`.

Comment (commitay) [inline Casks/tomtom-mydrive-connect.rb]: It will be removed by the other two lines in `uninstall pkgutil`

**Comment by fturcheti** [comment]
Thanks for the fix, @commitay.

**Comment by miccal** [comment]
Thank you for your contribution.

**Pull request merged**